pipelines/scheduled/generate-core-data.yml

```diff
@@ -0,0 +1,207 @@
+                  targetType: 'inline'
+                  script: |
+                    set -euo pipefail
+                    git clone https://$GITHUB_TOKEN@github.com/dotnet/website.git dotnet-website
+
+              - task: Bash@3
+                displayName: 'Update Website Repository with New Thanks Data'
+                condition: eq(variables['HAS_CORE_JSON'], 'true')
+                env:
+                  GITHUB_TOKEN: $(GITHUB_TOKEN)
+                  BUILD_ID: $(Build.BuildId)
+                  CORE_JSON_PATH: $(CORE_JSON_PATH)
+                inputs:
+                  targetType: 'inline'
+                  script: |
+                    set -euo pipefail
+                    
+                    cd dotnet-website
+                    
+                    # Configure git
+                    git config --global user.email "bot@azuredevops.com"
+                    git config --global user.name "azure-pipelines"
+                    git remote set-url origin "https://$GITHUB_TOKEN@github.com/dotnet/website.git"
+
+                    # Sync with remote
+                    git fetch origin main
+                    git checkout main
+                    git reset --hard origin/main
+
+                    # Create and switch to new branch
+                    branch="thanks-data-update-$BUILD_ID"
+                    git checkout -B "$branch"
+
+                    # Copy the updated core.json file AFTER git reset
+                    target_dir="website-resources/blob-assets/json/thanks"
+                    echo "Source file: $CORE_JSON_PATH"
+                    echo "Target directory: $target_dir"
+                    
+                    # Verify source file exists
+                    if [ ! -f "$CORE_JSON_PATH" ]; then
+                        echo "Source file not found: $CORE_JSON_PATH"
+                        exit 1
+                    fi
+                    
+                    # Create target directory and copy file
+                    mkdir -p "$target_dir"
+                    cp "$CORE_JSON_PATH" "$target_dir/core.json"
+                    echo "Successfully copied core.json to $target_dir/core.json"
+
+                    # Stage changes
+                    git add "website-resources/blob-assets/json/thanks/core.json"
+                    
+                    # Check if there are changes to commit
+                    if git diff --cached --quiet; then
+                        echo "No changes to commit"
+                        exit 0
+                    fi
+                    
+                    # Commit and push
+                    git commit -m "Update .NET thanks data"
+                    git push origin "HEAD:$branch"
+
+                    echo "##vso[task.setvariable variable=BRANCH_NAME]$branch"
+                    echo "Branch '$branch' created and pushed successfully"
+
+              - task: Bash@3
+                displayName: 'Create PR for .NET Thanks Data Update'
```

Comment:
Can we try to use ` - task: ShaykiAbramczyk.CreatePullRequest.CreatePullRequest.CreatePullRequest@1` to create a new pull request so it doesn't create it as me?
https://github.com/dotnet/website/blob/2028c7d2008245d97cd2402c4c253e711313d610/website/pipelines/scheduled/update-resources.yml#L70